### PR TITLE
Add plugins config and only show "Add" if "proposalParams" is set to …

### PR DESCRIPTION
**Pull request by samuveth**

### CODE REVIEW

File: src/components/Modal/ProposalPlugins.vue [hunk 1]
```diff
@@ -79,8 +85,13 @@ export default {
       return `https://raw.githubusercontent.com/snapshot-labs/snapshot.js/master/src/plugins/${plugin}/logo.png`;
     }
 
-    function showButton(plugin) {
-      return plugin.name !== 'SafeSnap';
+    function showButton(pluginObj) {
```
Comment (bonustrack) [inline src/components/Modal/ProposalPlugins.vue:88]: You can hide the full block, not just the button

Comment (samuveth) [reply src/components/Modal/ProposalPlugins.vue:88]: Hm, are you sure we don't want some way to see the plugins that will be used on proposal creation?

Comment (samuveth) [reply src/components/Modal/ProposalPlugins.vue:88]: If so I would need to redesign a lot of this component

**Comment by bonustrack** [comment]
@samuveth Can you resolve conflict then we can merge it

**Comment by samuveth** [comment]
@bonustrack I need your approval so I can merge.

**Review by bonustrack — APPROVED** [review]

**Pull request merged**